EIPS/eip-FOCIL.md

```diff
@@ -40,12 +40,55 @@ TBD
 A set of validators is selected from the beacon committee to become IL committee for `slot N`.
 
 - **`Slot N`, `t=0 to 8s`**: After processing the block for `slot N` and confirming it as the head, each IL committee member of `slot N` constructs a local inclusion list based on the head and their view of the public mempool, then broadcasts it over the P2P network.
-- **`Slot N`, `t=9s`**: IL committee members freeze their view of a set of local inclusion lists and no longer produce new ones.
-- **`Slot N`, `t=9 to 11s`**: IL committee members continue forwarding the local inclusion lists they are aware of but ignore any new ones. The block proposer and attesters of `slot N+1` continue listening to gossiped local inclusion lists. To ensure none are omitted, the block proposer can request for any missing local inclusion lists from a specific IL committee member via an RPC endpoint, for example, at `t=10s`.
-- **`Slot N`, `t=11s`**: The block proposer freezes its local inclusion lists view and IL committee members stop gossiping.
+- **`Slot N`, `t=9s`**: IL committee members freeze their view of a set of local ILs and no longer produce new ones.
+- **`Slot N`, `t=9 to 11s`**: IL committee members continue forwarding the local ILs they are aware of but ignore any new ones. The block proposer and attesters of `slot N+1` continue listening to gossiped local inclusion lists. To ensure none are omitted, the block proposer can request for any missing local inclusion lists from a specific IL committee member via an RPC endpoint, for example, at `t=10s`.
+- **`Slot N`, `t=11s`**: The block proposer freezes its local ILs view and IL committee members stop gossiping.
```

Comment:
It may be worth mentioning that at 11s, the block proposer will update its block with local ILs. Up to you.